app/controllers/event_instances_controller.rb

```diff
@@ -16,8 +16,9 @@ def update
   end
 
   def index
-    @event_instances = (params[:live] == 'true') ? EventInstance.live : EventInstance.latest
-    render partial: 'hangouts' if request.xhr?
```

Comment:
I remove line 20, because on js request, I want to respond with ```views/event_instances/index.js.erb```. I think it was dead code.